Cargo.toml

```diff
@@ -16,22 +16,17 @@ prost-codec = ["prost", "prost-derive", "lazy_static", "protobuf-build/grpcio-pr
 
 [dependencies]
 protobuf = "=2.8.0"
-prost = { version = "0.7", optional = true }
-prost-derive = { version = "0.7", optional = true }
+prost = { version = "0.9", optional = true }
+prost-derive = { version = "0.9", optional = true }
 futures = "0.3.5"
 raft-proto = { version = "0.7.0", default-features = false }
-grpcio = { version = "0.9.0", default-features = false, features = ["secure"] }
+grpcio = { version = "0.10.0", default-features = false, features = ["openssl-vendored"] }
```

Comment:
Why add `openssl-vendored` in kvproto? To my understanding, the feature replacing `secure` is `boringssl`.